values.yaml

```diff
@@ -81,14 +77,13 @@ libpostal:
   nodeSelector: {}
 
 interpolation:
-  enabled: false
+  enabled: true
```

Comment:
keep original. overriding values will be in another file that should not be committed to this repo